fix(kic): fix kubernetes ingress controller docs (#5230)
Kong/docs.konghq.com: app/_src/kubernetes-ingress-controller/references/cli-arguments.md

| `--proxy-sync-seconds`                   | `float32`          | Define the rate (in seconds) in which configuration updates will be applied to the Kong Admin API.                                                  | `3`                               |
| `--proxy-timeout-seconds`                | `float32`          | Define the rate (in seconds) in which the timeout configuration will be applied to the Kong client.                                                 | `10`                              |
| `--publish-service`                      | `string`           | Service fronting Ingress resources in "namespace/name" format. The controller will update Ingress status information with this Service's endpoints. |                                   |
| `--publish-status-address`               | `strings`          | User-provided addresses in comma-separated string format, for use in lieu of "publish-service" when that Service lacks useful address information.  |                                   |

{% if_version gte:2.9.x %}

> | `--publish-service-udp`                  | `string`           | Service fronting UDP routing resources in "namespace/name" format. The controller will update Ingress status information with this Service's endpoints. If omitted, the same Service will be used for both TCP and UDP routes. |                                   |

Reviewer: ```suggestion
| `--publish-service-udp`                  | `string`           | Service fronting UDP routing resources in "namespace/name" format. The controller updates Ingress status information with this Service's endpoints. If omitted, the same Service is used for both TCP and UDP routes. |                                   |
```

> | `--publish-status-address-udp`           | `strings`          | User-provided addresses in comma-separated string format, for use in lieu of "publish-service-udp" when that Service lacks useful address information. |                                   |

Reviewer: ```suggestion
| `--publish-status-address-udp`           | `strings`          | User-provided addresses in comma-separated string format, for use instead of `publish-service-udp` when that Service lacks useful address information. |                                   |
```
Switched `in lieu of` to `instead of` for simplicity + clarity for non-native speakers.
